libs/dictionaries/src/lib/dictionaries/en/common.json

```diff
@@ -45,11 +45,11 @@
                 "title": "Information Models"
             },
             "servicesEvents": {
-                "description": "Overview of services, procedures, and events used when organizations process data",
+                "description": "Services provided on behalf of the public sector and events that may trigger such services",
```

Comment:
"Services provided by or on behalf of the public sector..." 
provided vs. offered ? Ser at jeg har brukt "offered" i ingressen til beskrivelsen til katalogen, men begge kan vel brukes.